src/components/_custom/input-rodne-cislo/intput-rodne-cislo.js

```diff
@@ -0,0 +1,32 @@
+import { nodeListForEach } from "../../../common";
+import "../../../vendor/polyfills/Function/prototype/bind";
+import "../../../vendor/polyfills/Element/prototype/classList";
+import {parseRC, isValidRC} from "./input-rodne-cislo-parse"
+
+function SdnInputRodneCislo($module) {
```

Comment:
Preco nie class ?

Reply:
Toto bol format ktory bol pouzivany v starsom kode, prilis som sa nad tym nezamyslal. A nie som si isty, ci to ten ich preprocessor zvlada.